app/components/ProjectRevision/RevisionStatusWidget.tsx

```diff
@@ -52,6 +56,7 @@ const RevisionStatusWidget: React.FC<WidgetProps> = (props) => {
           },
         },
       },
+      onCompleted: () => router.replace(getProjectRevisionViewPageRoute(id)),
```

Comment:
Nice. I haven't used `replace` before but it makes sense here so the user can't use the back button to undo changes